src/components/FilterBar.vue

```diff
@@ -0,0 +1,137 @@
+<script setup lang="ts">
+import { FilterType } from "../types/filter";
+
+const objektClass = defineModel<string | null>("objektClass", {
+  required: true,
+});
+const objektSeason = defineModel<string | null>("objektSeason", {
+  required: true,
+});
+const group = defineModel<string | null>("group", { required: true });
+const artist = defineModel<string | null>("artist", { required: true });
+
+const tripleSMembers = [
+  "SeoYeon",
+  "HyeRin",
+  "JiWoo",
+  "ChaeYeon",
+  "YooYeon",
+  "SooMin",
+  "NaKyoung",
+  "YuBin",
+  "Kaede",
+  "DaHyun",
+  "Kotone",
+  "YeonJi",
+  "Nien",
+  "SoHyun",
+  "Xinyu",
+  "Mayu",
+  "Lynn",
+  "JooBin",
+  "HaYeon",
+  "ShiOn",
+  "ChaeWon",
+  "Sullin",
+  "SeoAh",
+  "JiYeon",
+];
+
+const tripleSSubunits = [
+  "AAA",
+  "EVOLution",
+  "Glow",
+  "KRE",
+  "LOVELution",
+  "NXT",
+];
+
+const tripleSArtists = [...tripleSMembers, ...tripleSSubunits];
+
+const artmsMembers = ["HeeJin", "HaSeul", "KimLip", "JinSoul", "Choerry"];
+const artmsSubunits = ["OEC"];
+
+const artmsArtists = [...artmsMembers, ...artmsSubunits];
+</script>
+
+<template>
+  <div class="row">
+    <div class="col">
+      <div class="fieldset">
+        <label for="objektClass">Class</label>
+        <select id="objektClass" v-model="objektClass">
+          <option :value="null">All</option>
+          <option value="First">First</option>
+          <option value="Special">Special</option>
+          <option value="Welcome">Welcome</option>
+          <option value="Double">Double</option>
+          <option value="Zero">Zero</option>
```

Comment:
Adding an option for Premier Class Objekts?

Reply:
Same question as above for this one.